apps/protocol-frontend/src/components/MyAttestationsTable.tsx

```diff
@@ -137,28 +138,33 @@ const MyAttestationsTable = ({
         >
           <Table {...getTableProps()} maxWidth="100vw" overflowX="auto">
             <Thead backgroundColor="gray.50">
-              {headerGroups.map((headerGroup: any) => (
-                <Tr {...headerGroup.getHeaderGroupProps()}>
-                  {headerGroup.headers.map((column: any) => (
-                    <Th
-                      {...column.getHeaderProps(column.getSortByToggleProps())}
-                      isNumeric={column.isNumeric}
-                      borderColor="gray.100"
-                    >
-                      {column.render('Header')}
-                      <chakra.span paddingLeft="4">
-                        {column.isSorted ? (
-                          column.isSortedDesc ? (
-                            <IoArrowDown aria-label="sorted-descending" />
-                          ) : (
-                            <IoArrowUp aria-label="sorted-ascending" />
-                          )
-                        ) : null}
-                      </chakra.span>
-                    </Th>
-                  ))}
-                </Tr>
-              ))}
+              {headerGroups.map(
+                (headerGroup: HeaderGroup<MyAttestationsTableType>) => (
```

Comment:
Well Done!

But I am not noticing we have like many of this: same structure, same style but different data. A separate component could handle this precedents. right?

Reply:
This pr is purely for removing linting issues and enforcing the current state of our linter warnings in the build. It isn't meant to go beyond that such as changing functionality.

Reply:
But, I totally aggree a lot of these headers and tables can be mode more reusable and modular. We can definitely handle in another ticket